packages/dashboard/src/components/confirmModal/confirmModal.css

```diff
@@ -0,0 +1,9 @@
+.btn-custom-primary {
```

Comment:
we should not be styling cloudscape components

Reply:
Yeah, but the requirement asking us to give the orange(#ff9900) background color for the button. 

![Uploading image.png…]()

Reply:
talked to Navita and this is the AWS primary button color which is not found in our version of cloudscape
not a blocker for now